assembl/static2/js/app/components/debate/brightMirror/sideComment/sideCommentBox.jsx

```diff
@@ -242,6 +245,33 @@ class DumbSideCommentBox extends React.Component<Props, State> {
     });
   };
 
+  editPost = () => {
+    const { contentLocale, refetchPost, updatePost } = this.props;
+    const { body, editingPostId } = this.state;
+
+    const bodyIsEmpty = editorStateIsEmpty(body);
+    if (bodyIsEmpty) {
+      displayAlert('warning', I18n.t('debate.thread.fillBody'));
+    }
+
+    const postVars = {
```

Comment:
is this code executed if the body is empty? if yes then you have to add an 'else'

Reply:
I forgot the return statement